### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by gabzn**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import Login from './components/Login';
 import Signup from './components/Signup';
 import Users from './components/Users';
 import Alert from './components/Alert';
+import Dashboard from './components/Dashboard';
```
Comment (gabzn) [inline client/src/App.js:11]: Added dashboard page.

File: client/src/App.js [hunk 2]
```diff
@@ -25,6 +26,7 @@ const App = () => {
 					<Route path='/' element={<Home />} />
 					<Route path='/login' element={<Login />} />
 					<Route path='/signup' element={<Signup />} />
+					<Route path='/dashboard' element={<Dashboard/>} />
```
Comment (gabzn) [inline client/src/App.js:29]: added dashboard route.

File: client/src/components/Login.js [hunk 3]
```diff
@@ -12,50 +12,62 @@ import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
...
+import axios from 'axios';
 
 const theme = createTheme();
 
 function Login() {
   const [email, setEmail] = useState('')
 	const [password, setPassword] = useState('')
 
-  const handleSubmit = (event) => {
+  async function loginUser(event) {
```
Comment (gabzn) [inline client/src/components/Login.js:24]: Rewrote loginUser function.

File: server/routes/authRoute.js [hunk 4]
```diff
@@ -40,21 +40,12 @@ router.post(
 		try {
 			/* See if user exists */
 			let user = await User.findOne({ email });
-
-			if (!user) {
-				return res
-					.status(400)
-					.json({ errors: 'Invalid Credentials'});
-			}
+			if (!user) return res.json({ error: 'Invalid Credentials'});
```
Comment (gabzn) [inline server/routes/authRoute.js:43]: Got rid of status code because of error handling.

File: server/routes/authRoute.js [hunk 5]
```diff
@@ -40,21 +40,12 @@ router.post(
...
 			/* Check if the user matches the password */
 			const isMatch = await bcrypt.compare(password, user.password);
-
-			if (!isMatch) {
-				return res
-					.status(400)
-					.json({ errors: 'Invalid Credentials'});
-			}
-
+			if (!isMatch) return res.json({ error: 'Invalid Credentials'});
```
Comment (gabzn) [inline server/routes/authRoute.js:47]: Got rid of status code because of error handling.

File: server/routes/authRoute.js (hunk 4 above)
Comment (channnx) [reply server/routes/authRoute.js:43]: Should we keep 400 status?

Comment (gabzn) [reply server/routes/authRoute.js:43]: Keep this one for now. I need it for testing.

**Review by channnx — APPROVED** [review]

**Pull request merged**